app.py

```diff
@@ -1,12 +1,36 @@
 #!/usr/bin/env python
-from flask import Flask, render_template, Response, request
-
+from flask import Flask, render_template, Response, request, flash
+from flask_sqlalchemy import SQLAlchemy
 import mediapipe_processor
+import psycopg2
```

Comment:
Edit the `requirements.txt` if the dependency has been changed.

Reply:
Follow the [PEP8#imports](https://peps.python.org/pep-0008/#imports) import dependencies.